src/PathCost.py

```diff
@@ -15,41 +16,41 @@ def __init__(self, par_state: State) -> None:
 
         for piece in par_state.board:
             match piece:
-                case chess_enums.Piece.W_P:
+                case Piece.W_P:
                     w_pieces += 1
                     break
-                case chess_enums.Piece.W_N:
+                case Piece.W_N:
                     w_pieces += 2
                     break
-                case chess_enums.Piece.W_B:
+                case Piece.W_B:
                     w_pieces += 3
                     break
-                case chess_enums.Piece.W_R:
+                case Piece.W_R:
                     w_pieces += 5
                     break
-                case chess_enums.Piece.W_Q:
+                case Piece.W_Q:
                     w_pieces += 9
                     break
-                case chess_enums.Piece.B_P:
+                case Piece.B_P:
                     b_pieces += 1
                     break
-                case chess_enums.Piece.B_N:
+                case Piece.B_N:
                     b_pieces += 2
                     break
-                case chess_enums.Piece.B_B:
+                case Piece.B_B:
                     b_pieces += 3
                     break
-                case chess_enums.Piece.B_R:
+                case Piece.B_R:
                     b_pieces += 5
                     break
-                case chess_enums.Piece.B_Q:
+                case Piece.B_Q:
                     b_pieces += 9
                     break
                 case _:
                     break
 
         self.piece_weight = (
-            w_pieces / b_pieces) if player == chess_enums.Player.White else (b_pieces / w_pieces)
+            w_pieces / b_pieces) if player == Player.White else (b_pieces / w_pieces)
```

Comment:
Can you do this in Python? (have the body of the if statement before the condition)

Reply:
https://www.geeksforgeeks.org/ternary-operator-in-python/

This is python's ternary operator

Reply:
This syntax is weird to me. I miss the js, c++ `isTrue ? this : that` syntax